Jenkinsfile

```diff
@@ -70,7 +70,9 @@ pipeline {
             post {
                 always {
                     archiveArtifacts "_output/tests/**/*"
-                    junit "_output/tests/**/unit-tests.xml"
+                    // NOTE(hasheddan): this should be re-enabled when unit
+                    // tests exist in this repo
+                    // junit "_output/tests/**/unit-tests.xml"
```

Comment:
@negz this was added to make this pass CI. It was failing because there are no unit tests. Disabling only this line means that if unit tests are needed in the future they will still be run automatically, even if we forget to re-enable this line.